src/omv/imlib/imlib.h

```diff
@@ -753,6 +753,13 @@ bool image_get_mask_pixel(image_t *ptr, int x, int y);
 #define JPEG_422_YCBCR_MCU_SIZE    ((JPEG_444_GS_MCU_SIZE) * 4)
 #define JPEG_420_YCBCR_MCU_SIZE    ((JPEG_444_GS_MCU_SIZE) * 6)
 
+typedef enum jpeg_subsampling {
+    JPEG_SUBSAMPLING_AUTO = 0,
+    JPEG_SUBSAMPLING_444  = 1, // Chroma subsampling 4:4:4 (No subsampling)
+    JPEG_SUBSAMPLING_422  = 2, // Chroma subsampling 4:2:2
+    JPEG_SUBSAMPLING_420  = 3, // Chroma subsampling 4:2:0
```

Comment:
We should remove the old `jpeg_subsample_t`. Note those HV sampling values are written to SOF0, so we can either set the value here or fix in `jpeg_write_headers`.

```suggestion
    JPEG_SUBSAMPLING_444  = 0x11, // Chroma subsampling 4:4:4 (No subsampling)
    JPEG_SUBSAMPLING_422  = 0x21, // Chroma subsampling 4:2:2
    JPEG_SUBSAMPLING_420  = 0x22, // Chroma subsampling 4:2:0
```

Reply:
Done